versions-manifest.json

```diff
@@ -224,6 +269,51 @@
       }
     ]
   },
+  {
+    "version": "3.12.3",
+    "stable": true,
+    "release_url": "https://github.com/actions/python-versions/releases/tag/3.12.3-8625548520",
+    "files": [
+      {
+        "filename": "python-3.12.3-darwin-arm64.tar.gz",
+        "arch": "arm64",
+        "platform": "darwin",
+        "download_url": "https://github.com/actions/python-versions/releases/download/3.12.3-8625548520/python-3.12.3-darwin-arm64.tar.gz"
+      },
+      {
+        "filename": "python-3.12.3-darwin-x64.tar.gz",
+        "arch": "x64",
+        "platform": "darwin",
+        "download_url": "https://github.com/actions/python-versions/releases/download/3.12.3-8625548520/python-3.12.3-darwin-x64.tar.gz"
+      },
+      {
+        "filename": "python-3.12.3-linux-20.04-x64.tar.gz",
+        "arch": "x64",
+        "platform": "linux",
+        "platform_version": "20.04",
+        "download_url": "https://github.com/actions/python-versions/releases/download/3.12.3-8625548520/python-3.12.3-linux-20.04-x64.tar.gz"
+      },
+      {
+        "filename": "python-3.12.3-linux-22.04-x64.tar.gz",
+        "arch": "x64",
+        "platform": "linux",
+        "platform_version": "22.04",
+        "download_url": "https://github.com/actions/python-versions/releases/download/3.12.3-8625548520/python-3.12.3-linux-22.04-x64.tar.gz"
+      },
+      {
+        "filename": "python-3.12.3-win32-x64.zip",
+        "arch": "x64",
+        "platform": "win32",
+        "download_url": "https://github.com/actions/python-versions/releases/download/3.12.3-8625548520/python-3.12.3-win32-x64.zip"
+      },
+      {
+        "filename": "python-3.12.3-win32-x86.zip",
+        "arch": "x86",
+        "platform": "win32",
+        "download_url": "https://github.com/actions/python-versions/releases/download/3.12.3-8625548520/python-3.12.3-win32-x86.zip"
+      }
+    ]
+  },
```

Comment:
@HarithaVattikuti this wasn't included in the https://github.com/actions/python-versions/releases/tag/3.12.3-8625548520 release, so updated actions are failing:
![image](https://github.com/actions/python-versions/assets/2906988/62abc930-199f-4a6e-8639-88c6f9b0402a)

Reply:
Hi @TWiStErRob 
The PR is merged and the 3.12.3 version will be installed now. Please check

Reply:
All good, setup-python concluded successfully. Thanks!